eagle-external/eagle-cdh/src/aux/eagle-server.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
```

Comment:
Yes,it's in different format than eagle.conf, one is json file and another is key-value file. Please search eagle.properties in descriptor/service.sdl. Maybe, I can delete eagle_server.sh, and mention the change in README.md.